Homework18 - Jenniffer Acpsta

acostjen1978 opened the pull request:
Test- PlaySongTest - Passed

acostjen1978 committed:
Homework18 - Jenniffer Acpsta

Test- PlaySongTest - Passed

rajniss21 commented:
Hello @acostjen1978,

Greetings! I have reviewed your homework assignment 18 and noticed that you encountered an error during code compilation. I kindly request you to please review your code and make the necessary corrections.

Best regards,
HW team (Rajnish, rajnish@testpro.io)!

![Screenshot_2](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/25093177/ecbe869d-8fc8-4eea-8ab0-23e6beb3b1ce)

HINT:
- In the Homework18.java class, you have defined a test using `@Test (dataProvider = "CorrectLoginProviders", dataProviderClass = BaseTest.class)`, where the dataProviderClass is referred from the BaseTest class. However, the dataProvider class is not defined in the BaseTest.
- To resolve this, please remove the line of code: `@Test (dataProvider = "CorrectLoginProviders", dataProviderClass = BaseTest.class)`

![Screenshot_3](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/25093177/807a0df9-493f-4d7d-99cf-5685dbd91aea)

- The `enterEmail` method in the code already accepts the parameter `String email`. Therefore, you do not need to pass the same parameter from the Homework18 class. 
- To fix this, please remove the parameters `String email` and `password` and instead pass the values of email and password directly.

![Screenshot_5](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/25093177/47e60567-50bb-49a2-84e9-c841dfb0622e)

![Screenshot_4](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/25093177/4bea9667-55a3-41ea-9798-448cdd12efba)

acostjen1978 committed:
Resubmit- Homework18

I made changes you suggested and now I don't know what is wrong with my test. Help please.
